### PR TITLE
Added red and yellow VRFs and traffic test across hosts

### PR DESCRIPTION
**Pull request by venkatmahalingam**
Added red and yellow VRFs and traffic test across hosts.

**Test log:**
======
host2-leaf2_1      | + ip link add link eth0 name eth0.10 type vlan id 10
host2-leaf2_1      | + ip link set eth0 up
host2-leaf2_1      | + ip addr flush dev eth0
host2-leaf2_1      | + ip link set eth0.10 up
host2-leaf2_1      | + ip addr add 10.10.10.11/24 dev eth0.10
host2-leaf2_1      | + sleep infinity
host3-leaf2_1      | + ip link add link eth0 name eth0.30 type vlan id 30
host3-leaf2_1      | + ip link set eth0 up
host3-leaf2_1      | + ip addr flush dev eth0
host3-leaf2_1      | + ip link set eth0.30 up
host3-leaf2_1      | + ip addr add 30.30.30.31/24 dev eth0.30
host3-leaf2_1      | + ip route replace default via 30.30.30.1
host3-leaf2_1      | + sleep infinity
opi-test_1         | + ip link add link eth0 name eth0.10 type vlan id 10
opi-test_1         | + ip link set eth0 up
opi-test_1         | + ip addr flush dev eth0
opi-test_1         | + ip link set eth0.10 up
opi-test_1         | + ip addr add 10.10.10.10/24 dev eth0.10
opi-test_1         | + ping -c 3 10.10.10.11
opi-test_1         | PING 10.10.10.11 (10.10.10.11) 56(84) bytes of data.
opi-test_1         | 64 bytes from 10.10.10.11: icmp_seq=1 ttl=64 time=0.304 ms
opi-test_1         | 64 bytes from 10.10.10.11: icmp_seq=2 ttl=64 time=0.127 ms
opi-test_1         | 64 bytes from 10.10.10.11: icmp_seq=3 ttl=64 time=0.045 ms
opi-test_1         |
opi-test_1         | --- 10.10.10.11 ping statistics ---
opi-test_1         | 3 packets transmitted, 3 received, 0% packet loss, time 2055ms
opi-test_1         | rtt min/avg/max/mdev = 0.045/0.158/0.304/0.108 ms
opi-test_1         | + ip link add link eth1 name eth1.20 type vlan id 20
opi-test_1         | + ip link set eth1 up
opi-test_1         | + ip addr flush dev eth1
opi-test_1         | + ip link set eth1.20 up
opi-test_1         | + ip addr add 20.20.20.20/24 dev eth1.20
opi-test_1         | + ip route replace default via 20.20.20.1
opi-test_1         | + ping -c 3 30.30.30.31
opi-test_1         | PING 30.30.30.31 (30.30.30.31) 56(84) bytes of data.
opi-test_1         | 64 bytes from 30.30.30.31: icmp_seq=1 ttl=63 time=0.267 ms
opi-test_1         | 64 bytes from 30.30.30.31: icmp_seq=2 ttl=63 time=0.120 ms
opi-test_1         | 64 bytes from 30.30.30.31: icmp_seq=3 ttl=63 time=0.094 ms
opi-test_1         |
opi-test_1         | --- 30.30.30.31 ping statistics ---
opi-test_1         | 3 packets transmitted, 3 received, 0% packet loss, time 2037ms
opi-test_1         | rtt min/avg/max/mdev = 0.094/0.160/0.267/0.076 ms
opi-test_1         | + ping -c 3 5.5.5.5
opi-test_1         | PING 5.5.5.5 (5.5.5.5) 56(84) bytes of data.
opi-test_1         | 64 bytes from 5.5.5.5: icmp_seq=1 ttl=63 time=0.237 ms
opi-test_1         | 64 bytes from 5.5.5.5: icmp_seq=2 ttl=63 time=0.106 ms
opi-test_1         | 64 bytes from 5.5.5.5: icmp_seq=3 ttl=63 time=0.097 ms
opi-test_1         |
opi-test_1         | --- 5.5.5.5 ping statistics ---
opi-test_1         | 3 packets transmitted, 3 received, 0% packet loss, time 2046ms
opi-test_1         | rtt min/avg/max/mdev = 0.097/0.146/0.237/0.063 ms
opi-test_1         | + ip link add link eth2 name eth2.40 type vlan id 40
opi-test_1         | + ip link set eth2 up
opi-test_1         | + ip addr flush dev eth2
opi-test_1         | + ip link set eth2.40 up
opi-test_1         | + ip addr add 40.40.40.40/24 dev eth2.40
opi-test_1         | + ip route replace default via 40.40.40.1
opi-test_1         | + ping -c 3 6.6.6.6
opi-test_1         | PING 6.6.6.6 (6.6.6.6) 56(84) bytes of data.
opi-test_1         | 64 bytes from 6.6.6.6: icmp_seq=1 ttl=63 time=0.148 ms
opi-test_1         | 64 bytes from 6.6.6.6: icmp_seq=2 ttl=63 time=0.099 ms
opi-test_1         | 64 bytes from 6.6.6.6: icmp_seq=3 ttl=63 time=0.096 ms
opi-test_1         |
opi-test_1         | --- 6.6.6.6 ping statistics ---
opi-test_1         | 3 packets transmitted, 3 received, 0% packet loss, time 2038ms
opi-test_1         | rtt min/avg/max/mdev = 0.096/0.114/0.148/0.023 ms
opi-test_1         | + ip link add link eth3 name eth3.50 type vlan id 50
opi-test_1         | + ip link set eth3 up
opi-test_1         | + ip addr flush dev eth3
opi-test_1         | + ip link set eth3.50 up
opi-test_1         | + ip addr add 50.50.50.50/24 dev eth3.50
opi-test_1         | + ip route replace default via 50.50.50.1
opi-test_1         | + ping -c 3 7.7.7.7
opi-test_1         | PING 7.7.7.7 (7.7.7.7) 56(84) bytes of data.
opi-test_1         | 64 bytes from 7.7.7.7: icmp_seq=1 ttl=63 time=0.157 ms
opi-test_1         | 64 bytes from 7.7.7.7: icmp_seq=2 ttl=63 time=0.139 ms
opi-test_1         | 64 bytes from 7.7.7.7: icmp_seq=3 ttl=63 time=0.044 ms
opi-test_1         |
opi-test_1         | --- 7.7.7.7 ping statistics ---
opi-test_1         | 3 packets transmitted, 3 received, 0% packet loss, time 2038ms
opi-test_1         | rtt min/avg/max/mdev = 0.044/0.113/0.157/0.049 ms
opi-evpn-bridge_opi-test_1 exited with code 0

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -195,29 +359,92 @@ services:
...
+                ip link set eth1 up && ip addr flush dev eth1 && \
+                ip link set eth1.20 up && \
+                ip addr add 20.20.20.20/24 dev eth1.20 && \
+                ip route replace default via 20.20.20.1 && \
+                ping -c 3 30.30.30.31 && ping -c 3 5.5.5.5 && \
+                ip link add link eth2 name eth2.40 type vlan id 40 && \
+                ip link set eth2 up && ip addr flush dev eth2 && \
+                ip link set eth2.40 up && \
+                ip addr add 40.40.40.40/24 dev eth2.40 && \
+                ip route replace default via 40.40.40.1 && \
```
Comment (glimchb) [inline docker-compose.yml:431]: instead of changing default `ip route replace default via 40.40.40.1` can you just use `ping -I interface 6.6.6.6` ?

Comment (venkatmahalingam) [reply docker-compose.yml:431]: No, if we use ping 6.6.6.6 with src intf, the ARP will be sent for 6.6.6.6 from host, that's not correct.

Comment (glimchb) [reply docker-compose.yml:431]: I see, thanks

**Review by glimchb — APPROVED** [review]
looks good, we can iterate on this as needed

**Comment by glimchb** [comment]
@mardim91 check as well, please

**Pull request merged**